tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.psm1

```diff
@@ -1055,3 +1084,112 @@ function Set-BootstrapRepo
 	param([string]$Repo)
 	$script:BootStrapRepo = $Repo
 }
+
+<#
+.ExternalHelp help\AzureRM.Bootstrapper-help.xml 
+#>
+function Set-AzureRmDefaultProfile
+{
+  [CmdletBinding(SupportsShouldProcess = $true)]
+  param()
+  DynamicParam
+  {
+    $params = New-Object -Type System.Management.Automation.RuntimeDefinedParameterDictionary
+    Add-ProfileParam $params
+    Add-ForceParam $params
+    Add-ScopeParam $params
+    return $params
+  }
+  PROCESS {
+    $armProfile = $PSBoundParameters.Profile
+    $Scope = $PSBoundParameters.Scope
+    $Force = $PSBoundParameters.Force
+    
+    $defaultProfile = $Global:PSDefaultParameterValues["*-AzureRmProfile:Profile"]
+    if ($defaultProfile -ne $armProfile)
+    {
+      # Set DefaultProfile for this session
+      $Global:PSDefaultParameterValues["*-AzureRmProfile:Profile"]="$armProfile"
+
+      # Edit the profile content
+      $profileContent = @"
+          `$PSDefaultParameterValues["*-AzureRmProfile:Profile"]="$armProfile"
```

Comment:
right now, the default is being passed only for the *profile* cmdlets. agree that we need to change the default parameter for Import-Module as well. just talked to vishnu. it will be part of the next revision of this PR